lib/fhir/root.js

```diff
@@ -271,8 +272,9 @@ module.exports = (mongo, modules, callback) => {
     searchAll: (ctx) => {
       throw new Error('Not yet supported')
     },
-    conformance: (ctx) => {
-      throw new Error('Not yet supported')
+    conformance: (ctx, callback) => {
```

Comment:
This should be renamed now that FHIR has switched from the term "conformance" to "capability".